WalletWasabi.Fluent/ViewModels/Dialogs/InsufficientBalanceDialogViewModel.cs

```diff
@@ -7,25 +10,42 @@ namespace WalletWasabi.Fluent.ViewModels.Dialogs
 	[NavigationMetaData(Title = "Insufficient Balance")]
 	public partial class InsufficientBalanceDialogViewModel : DialogViewModelBase<bool>
 	{
-		public InsufficientBalanceDialogViewModel(BalanceType type)
+		public InsufficientBalanceDialogViewModel(BalanceType type, BuildTransactionResult transaction, decimal usdExchangeRate)
 		{
+			var destinationAmount = transaction.CalculateDestinationAmount().ToDecimal(MoneyUnit.BTC);
+			var fee = transaction.Fee;
+
+			BtcAmountText = $"{destinationAmount} bitcoins ";
+			FiatAmountText = $"(≈{(destinationAmount * usdExchangeRate).FormattedFiat()} USD) ";
+
+			BtcFeeText = $"{fee.ToDecimal(MoneyUnit.Satoshi)} satoshis ";
```

Comment:
should we really write out `satoshis` here?
I think the shorter `sats` could be used in the whole UI...

Reply:
I don't have an opinion about this. Which one is widely used?

Reply:
I think `sats` is shaping up to be more the standard.
But I'm also not sure which we should go with.
I guess a slight personal preference to `sats`.

anyhow, not in scope for this PR, so let's talk about it here... https://github.com/zkSNACKs/WalletWasabi/discussions/5374